wire-runtime/src/commonMain/kotlin/com/squareup/wire/ProtoAdapter.kt

```diff
@@ -514,7 +514,7 @@ internal class DoubleArrayProtoAdapter(
   private val originalAdapter: ProtoAdapter<Double>,
 ) : ProtoAdapter<DoubleArray>(
   LENGTH_DELIMITED,
-  LongArray::class,
+  DoubleArray::class,
```

Comment:
Thank you Assertj?? Some tests were passing before and not anymore. cc @JGulbronson 
Same below